tabs/osd.js

```diff
@@ -816,7 +823,8 @@ OSD.chooseFields = function () {
               if (semver.gte(CONFIG.apiVersion, "1.37.0")) {
                   OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
                     F.RTC_DATE_TIME,
-                    F.ADJUSTMENT_RANGE
+                    F.ADJUSTMENT_RANGE,
+                    F.REMAINING_TIME_ESTIMATE
```

Comment:
Sorry, but this is still wrong, `REMAINING_TIME_ESTIMATE` has to go above `RTC_DATE_TIME`, so that the resulting list in `OSD.constants.DISPLAY_FIELDS` has the same ordering as in https://github.com/betaflight/betaflight/blob/e7d2e84348512768d93e5ada827c485cc08d16f0/src/main/io/osd.h#L86-L88